planning/mission_planning/mission_planner/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ project(mission_planner)
 ### Compile options
 if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
It looks like the linter tests haven't been added to the CMakelist yet?

Reply:
Fixed in bca948a0de2c2e0faebfcdfe2bd695ffbf69ae93